.github/workflows/build-images.yml

```diff
@@ -356,10 +265,9 @@ jobs:
       uses: spilchen/switch-case-action@v2
       id: minimal_vertica_image
       with:
-        default: ghcr.io/${{ env.OWNER_LC }}/vertica-k8s:${{ github.sha }}-minimal
+        default: docker.io/vertica/vertica-k8s-private:latest-master
         conditionals-with-values: |
           ${{ inputs.minimal_vertica_image != '' }} => ${{ inputs.minimal_vertica_image }}
-          ${{ github.event_name == 'pull_request' }} => vertica-k8s:kind-minimal
 
     - name: Login to GitHub Container registry for non-PRs
       uses: docker/login-action@v2
```

Comment:
The check in the line below we can update now that the default image isn't ghcr.io
```
if: ${{ github.event_name != 'pull_request' && startsWith(inputs.minimal_vertica_image, 'ghcr.io') }}
```